src/med_bench/mediation.py

```diff
@@ -90,6 +90,9 @@ def mediation_IPW(y, t, m, x, trim, regularization=True, forest=False,
     int
             number of used observations (non trimmed)
     """
+    # check input
+    y, t, m, x = _check_input(y, t, m, x, setting='multidimensional')
```

Comment:
Just wondering: should this be doe in each estimator or handled upstream in the function that calls all of them ?

Reply:
Good question. I would say that in the light of the big refactors we discussed, we should leave it there and include it in the refactor of `get_estimation` (that will change name)? For now I don't know if it is generic enough with the `setting` parameter, wdyt?